Dockerfile

```diff
@@ -6,10 +6,15 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends wget lsb-release ca-certificates gnupg unzip \
         python ssh rsync curl \
         python-jinja2 \
-        python-sphinx \
-        php7.3-apcu php7.3-bcmath php7.3-cli php7.3-curl php7.3-fpm php7.3-gd php7.3-intl php7.3-mysql php7.3-xml php7.3-zip php7.3-mbstring && \
-    echo "memory_limit = 1024M" >> /etc/php/7.3/cli/php.ini && \
-    echo "date.timezone = UTC" >> /etc/php/7.3/cli/php.ini && \
+        python-sphinx && \
+    wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg && \
+    sh -c 'echo "deb https://packages.sury.org/php/ buster main" > /etc/apt/sources.list.d/php.list' && \
+    apt-get update && \
```

Comment:
It's the second *apt-get update* of the command. It would be better to do it only once IMO (so move the source list on top)

Reply:
I can't, I need wget before :)